docs/features/G01-no-account-privacy-first.md

```
@@ -1,57 +1,57 @@
 priority: P0
-status: Proposed
+status: Built
 tags: [privacy, no-account, local-first, url-state, trust]
 ---
 
 # No-Account, Privacy-First
 
-**Competitor verdict:** Partial gap — no-account exists in one *logger* (Nighp), but every wake-window *scheduler* requires an account. "No account, and it actually schedules" is unoccupied.
+**Competitor verdict:** Partial gap â€” no-account exists in one *logger* (Nighp), but every wake-window *scheduler* requires an account. "No account, and it actually schedules" is unoccupied.
 
 ## Problem / user need
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)